components/notifier/src/quality_time_notifier.py

```diff
@@ -5,28 +5,25 @@
 import os
 import traceback
 from datetime import datetime
-from typing import Dict, Final, NoReturn, cast
+from typing import Final, List, NoReturn, cast
 
 import aiohttp
 
-from destinations.ms_teams import build_notification_text, send_notification_to_teams
+import outbox
 from notifier_utilities.type import JSON, URL
 from strategies.changed_status import get_notable_metrics_from_json
 
 
 async def notify(log_level: int = None) -> NoReturn:
     """Notify our users periodically of the number of red metrics."""
```

Comment:
Notifications are not just red metrics, right?

Reply:
notifications are either a metric that becomes red or becomes white at the moment.

Reply:
Please change the comment accordingly.